src/toast/ops/sim_ground.py

```diff
@@ -92,6 +92,8 @@ class SimGround(Operator):
         help="Name of built-in weather site (e.g. 'atacama', 'south_pole') or path to HDF5 file",
     )
 
+    realization = Int(0, help="The realization index")
```

Comment:
Should we call this "weather_realization" or something to indicate what it is being used for?

Reply:
All operators producing randomized output should have a realization index trait. What that index refers to should be apparent from the context. Do you agree?

Reply:
I guess my point was that the the `SimGround` operator is simulating a lot of stuff:  telescope pointing, weather, etc.  Is the `realization` for the weather?  The telescope pointing?  The HWP angle?  I guess we could rename that if we are ever simulating different realizations of different types of data.  Or we could derive the realization indices for different things from one input realization number.